src/preferences/dialog/dlgprefwaveform.cpp

```diff
@@ -221,8 +221,8 @@ void DlgPrefWaveform::slotResetToDefaults() {
     // Don't normalize overview.
     normalizeOverviewCheckBox->setChecked(false);
 
-    // 30FPS is the default
-    frameRateSlider->setValue(30);
+    // 60FPS is the default
+    frameRateSlider->setValue(60);
```

Comment:
:+1:

Reply:
75FPS displays get info fashion now, we should probably expand the possible framerate to 120FPS soon, also variable refresh rates are a thing now and might be interesting for us (though probably only relevant for games that have exclusive access to a display device)